node/src/gql/middleware.ts

```diff
@@ -22,6 +22,8 @@ export default graphqlHTTP((req: Request, res: Response) => {
         graphiql: true,
         context: {
           user: user || null,
+          res: res,
```

Comment:
this is probably too broad and we should define a narrower interface for whatever we're doing in graphql, e.g.

```typescript
context: {
  getAuthAndRefreshTokens() : AuthAndRefreshTokens => {...}
  setAuthAndRefreshTokens: (authAndRefreshTokens) => {...}
}
```

Reply:
Please provide more details for that. I think what I did is alright though.